### PR TITLE
1198178: Adds pool option to remove, unsubscribe command

### PR DESCRIPTION
**Pull request by cnsnyder**
Updates tests for remove, and stubs

Updates remove pool option to work in the unregistered case.

### CODE REVIEW

File: src/subscription_manager/certdirectory.py [hunk 1]
```diff
@@ -314,6 +314,23 @@ def list_for_product(self, product_id):
                     entitlements.append(cert)
         return entitlements
 
+    def list_for_pool_id(self, pool_id):
+        """
+        Returns all entitlement certificates provided by the given
+        product ID.
```
Comment (wottop) [inline src/subscription_manager/certdirectory.py]: Pool ID ?

File: test/test_remove.py [hunk 2]
```diff
@@ -62,18 +67,21 @@ def test_unsubscribe_unregistered(self):
...
                 StubProductDirectory([]))
         cmd = managercli.RemoveCommand()
 
-        cmd.main(['remove', '--serial=%s' % ent1.serial, '--serial=%s' % ent3.serial])
+        cmd.main(['remove', '--serial=%s' % ent1.serial, '--serial=%s' % ent3.serial, '--pool=%s' % ent4.pool.id])
         self.assertTrue(cmd.entitlement_dir.list_called)
         self.assertTrue(ent1.is_deleted)
         self.assertFalse(ent2.is_deleted)
         self.assertTrue(ent3.is_deleted)
+        self.assertTrue(ent4.is_deleted)
```
Comment (wottop) [inline test/test_remove.py]: make stylish fail

needs newline at end of file

Comment (cnsnyder) [inline test/test_remove.py]: new line added in latest force push

**Comment by wottop** [comment]
ack

**Pull request merged**